hedera-node/docs/design/services/token-service/airdrops/airdrop-transaction.md

```diff
@@ -0,0 +1,208 @@
+# Airdrop Transaction
+
+## Purpose
+
+The essence of HIP-904 is introducing a missing functionality in the Hedera ecosystem that enables airdrop transactions. 
+Airdrops are frequently used operation in the web3 ecosystem and a lot of blockchain, dapp or DAO developers incentivise users to explore and use their products through it. Early adopters are ofter awarded with the so called airdrops which are native tokens for the system and a dedicated amount of them are distributed in airdrops. Hedera could benefit from introducing this functionality and this is something longed for from the whole community.
```

Comment:
This reads better with some line wrapping (which will be removed in the rendered view), and there are a couple misspelled words and awkward grammar.
```suggestion
Airdrops are frequently used operation in the web3 ecosystem and a lot of blockchains,
 dapps, or DAO developers incentivize users to explore and use their products with
 this process. Early adopters are often awarded with so-called airdrops which are
 tokens for the system and a dedicated amount of them are distributed in such airdrops.
 Hedera will benefit from introducing this functionality and this is something long
 requested by the community.
```

Reply:
Noted